integration-tests/emulated/tests/collectives/collectives-polkadot/src/tests/fellowship_treasury.rs

```diff
@@ -0,0 +1,285 @@
+		type Balances = <AssetHubPolkadot as AssetHubPolkadotPallet>::Balances;
+
+		// Ensure that the funds deposited to the Fellowship Treasury account.
+
+		let fellowship_treasury_location: Location =
+			Location::new(1, [Parachain(1001), PalletInstance(65)]);
+		let fellowship_treasury_account =
+			AssetHubLocationToAccountId::convert_location(&fellowship_treasury_location).unwrap();
+
+		assert_eq!(
+			<Balances as Inspect<_>>::balance(&fellowship_treasury_account),
+			fellowship_treasury_balance
+		);
+
+		// Assert events triggered by xcm pay program:
+		// 1. treasury asset transferred to spend beneficiary;
+		// 2. response to Relay Chain Treasury pallet instance sent back;
+		// 3. XCM program completed;
+		assert_expected_events!(
+			AssetHubPolkadot,
+			vec![
+				RuntimeEvent::Balances(pallet_balances::Event::Transfer { .. }) => {},
+				RuntimeEvent::ParachainSystem(cumulus_pallet_parachain_system::Event::UpwardMessageSent { .. }) => {},
+				RuntimeEvent::MessageQueue(pallet_message_queue::Event::Processed { success: true ,.. }) => {},
+			]
+		);
+	});
+
+	CollectivesPolkadot::execute_with(|| {
+		type RuntimeEvent = <CollectivesPolkadot as Chain>::RuntimeEvent;
+		type RuntimeCall = <CollectivesPolkadot as Chain>::RuntimeCall;
+		type RuntimeOrigin = <CollectivesPolkadot as Chain>::RuntimeOrigin;
+		type Runtime = <CollectivesPolkadot as Chain>::Runtime;
+		type FellowshipTreasury =
+			<CollectivesPolkadot as CollectivesPolkadotPallet>::FellowshipTreasury;
+
+		// Spend assets from Fellowship Treasury.
+
+		let fellows_origin: RuntimeOrigin =
+			collectives_polkadot_runtime::fellowship::pallet_fellowship_origins::Origin::Fellows
+				.into();
+
+		// Set up an asset rate for the assets that will be spent.
+		// TODO: Ensure asset rate identifies relative native asset locations and maintains a 1:1
+		// rate without the need to set up a rate.
+
+		let asset_hub_location: Location = (Parent, Parachain(1000)).into();
+		let native_asset_on_asset_hub = Location::parent();
+
+		let asset_rate_call = RuntimeCall::AssetRate(pallet_asset_rate::Call::<Runtime>::create {
+			asset_kind: bx!(VersionedLocatableAsset::V4 {
+				location: asset_hub_location.clone(),
+				asset_id: native_asset_on_asset_hub.clone().into(),
+			}),
+			rate: 1.into(),
+		});
+
+		assert_ok!(asset_rate_call.dispatch(fellows_origin.clone()));
+
+		// Fund Alice account from Fellowship Treasury.
+
+		let alice_location: Location =
+			[Junction::AccountId32 { network: None, id: Polkadot::account_id_of(ALICE).into() }]
+				.into();
+
+		let fellowship_treasury_spend_call =
+			RuntimeCall::FellowshipTreasury(pallet_treasury::Call::<Runtime, Instance1>::spend {
+				asset_kind: bx!(VersionedLocatableAsset::V4 {
+					location: asset_hub_location,
+					asset_id: native_asset_on_asset_hub.into(),
+				}),
+				amount: fellowship_spend_balance,
+				beneficiary: bx!(VersionedLocation::V4(alice_location)),
```

Comment:
Same comment applies everywhere you're using explicit XCM version:

use from()/into() instead of hardcoding actual version so we don't have to change the tests when upgrading XCM version

```suggestion
				beneficiary: bx!(alice_location.into()),
```